src/formatting/expr.rs

```diff
@@ -124,6 +124,9 @@ pub(crate) fn format_expr(
         | ast::ExprKind::Loop(..)
         | ast::ExprKind::While(..) => to_control_flow(expr, expr_type)
             .and_then(|control_flow| control_flow.rewrite(context, shape)),
+        ast::ExprKind::ConstBlock(ref anon_const) => {
+            Some(format!("const {}", anon_const.rewrite(context, shape)?))
+        }
```

Comment:
This is new syntax so needed to add something here to handle the variant. This will work sufficiently well, but will open a new issue to track handling comments between the keyword and block open, add test cases, honor `brace_style`, etc.